packages/turbo/schema.json

```diff
@@ -0,0 +1 @@
nfiguration#cache","default":true},"inputs":{"type":"array","items":{"type":"string"},"description":"The set of glob patterns to consider as inputs to this task.\n\nChanges to files covered by these globs will cause a cache miss and the task will be rerun.\n\nIf a file has been changed that is **not** included in the set of globs, it will not cause a cache miss.\n\nIf omitted or empty, all files in the package are considered as inputs.\n\nDocumentation: https://turbo.build/repo/docs/reference/configuration#inputs","default":[]},"outputMode":{"$ref":"#/definitions/OutputMode","description":"Output mode for the task.\n\n\"full\": Displays all output\n\n\"hash-only\": Show only the hashes of the tasks\n\n\"new-only\": Only show output from cache misses\n\n\"errors-only\": Only show output from task failures\n\n\"none\": Hides all task output\n\nDocumentation: https://turbo.build/repo/docs/reference/command-line-reference#--output-logs","default":"full"},"persistent":{"type":"boolean","description":"Indicates whether the task exits or not. Setting `persistent` to `true` tells turbo that this is a long-running task and will ensure that other tasks cannot depend on it.\n\nDocumentation: https://turbo.build/repo/docs/reference/configuration#persistent","default":false}},"additionalProperties":false},"OutputMode":{"type":"string","enum":["full","hash-only","new-only","errors-only","none"]},"RemoteCache":{"type":"object","properties":{"signature":{"type":"boolean","description":"Indicates if signature verification is enabled for requests to the remote cache. When `true`, Turborepo will sign every uploaded artifact using the value of the environment variable `TURBO_REMOTE_CACHE_SIGNATURE_KEY`. Turborepo will reject any downloaded artifacts that have an invalid signature or are missing a signature.","default":false}},"additionalProperties":false},"WorkspaceSchema":{"type":"object","properties":{"$schema":{"type":"string","default":"https://turbo.build/schema.json"},"pipeline":{"type":"object","additionalProperties":{"$ref":"#/definitions/Pipeline","description":"The name of a task that can be executed by turbo. If turbo finds a workspace package with a package.json scripts object with a matching key, it will apply the pipeline task configuration to that npm script during execution."},"description":"An object representing the task dependency graph of your project. turbo interprets these conventions to schedule, execute, and cache the outputs of tasks in your project.\n\nDocumentation: https://turbo.build/repo/docs/reference/configuration#pipeline","default":{}},"extends":{"type":"array","items":{"type":"string"},"description":"This key is only available in Workspace Configs and cannot be used in your root turbo.json.\n\nTells turbo to extend your root `turbo.json` and overrides with the keys provided in your Workspace Configs.\n\nCurrently, only the \"//\" value is allowed.","default":["//"]}},"required":["extends","pipeline"],"additionalProperties":false}}}
```

Comment:
The lack of formatting was copied from `docs/public/schema.json` unsure if this is intentional or not.